src/jupytext/contentsmanager.py

```diff
@@ -387,7 +388,12 @@ def new_untitled(self, path="", type="", ext=""):
             We override the base function because that one does not take the 'ext' argument
             into account when type=="notebook". See https://github.com/mwouts/jupytext/issues/443
             """
-            if type != "notebook" and ext != ".ipynb":
+            # We are excluding all the cases that come up with Jupytext and using
+            # base method to create file. Is there a better way to get all the
+            # extensions that Jupytext uses so that we dont hard code them
+            # (even with regex) as below?
+            #
+            if not (type == "notebook" and re.match(r"[\.py\:\w+ | \.md]", ext)):
```

Comment:
@mwouts Is there a better way to check for all `jupytext` supported formats instead of using a regex? The idea here is to use original method for cases excluding `jupytext`.

Reply:
That's a good point. Yes, the current implementation works pretty well. During the development I modified the current implementation and never tested original implementation after. I will revert back to existing implementation. Cheers!